examples/base_difference_mapcubes.py

```diff
@@ -0,0 +1,41 @@
+"""
+========================================
+Base Difference in Mapcubes
+========================================
+
+"""
+############################################################################
+
+# Importing Libraries
+import numpy as np
+import sunpy.map as sm
```

Comment:
I think we have decided long ago the stule to just `import sunpy.map` without shorting it into anything.